src/middleware/authenticate.js

```diff
@@ -0,0 +1,37 @@
+const jwt = require("jsonwebtoken");
+const asyncHandler = require("express-async-handler");
+const { throwError } = require("../lib/errors");
+const { decodeJwt } = require("../lib/utils");
+
+const authenticate = async (req, res, next) => {
+  let token;
+  try {
+    let bearerHeader = req.headers["authorization"];
+    token = bearerHeader.split(" ")[1];
+    if (!bearerHeader) {
+      message = "No authentication token provided";
```

Comment:
Preceding this line with let keyword to declare message variable will resolve Codacy Quality PR